src/impls.rs

```diff
@@ -267,13 +268,13 @@ const _: () = unsafe {
 //   purpose of those types, it's virtually unthinkable that that would ever
 //   change. The only valid alignment for a 1-byte type is 1.
 //
-// FIXME(#429): Add quotes from documentation.
+// [1] Per https://doc.rust-lang.org/1.81.0/std/num/type.NonZeroU8.html:
+//
+//     `Option<NonZeroU8>` is guaranteed to be compatible with `u8`, including in FFI.
```

Comment:
Here (and below), it's probably also worth quoting the next line of the documentation:

> Thanks to the [null pointer optimization](https://doc.rust-lang.org/1.81.0/std/option/index.html#representation), NonZeroU8 and Option<NonZeroU8> are guaranteed to have the same size and alignment: